src/lib/utils.ts

```diff
@@ -33,6 +34,78 @@ export const getNodeTextContent = (node: ManuscriptNode) => {
   return textContent
 }
 
+export const getContributorTextContent = (
+  node: ManuscriptNode,
+  oldAttrs: Record<any, any> | undefined
+) => {
+  let textContent = ''
+  textContent += `${node.attrs.bibliographicName.given} ${node.attrs.bibliographicName.family}`
+  if (oldAttrs) {
+    textContent =
+      `${oldAttrs.bibliographicName.given} ${oldAttrs.bibliographicName.family} to ` +
+      textContent
+  }
+  return textContent
+}
+
+export const getAffiliationTextContent = (
+  node: ManuscriptNode,
+  oldAttrs: Record<any, any> | undefined
+) => {
+  let textContent = ''
+  textContent += `${node.attrs.institution}`
+  if (oldAttrs) {
+    textContent = `${oldAttrs.institution} to ` + textContent
+  }
+  return textContent
+}
+
+export const findBibliographyById = (
```

Comment:
Can you point me to where this is used? I can't find any usage.

Reply:
deleted